packages/@aws-cdk/aws-glue-alpha/README.md

```diff
@@ -223,7 +223,25 @@ new glue.Table(this, 'MyTable', {
 });
 ```
 
-By default, an S3 bucket will be created to store the table's data and stored in the bucket root. You can also manually pass the `bucket` and `s3Prefix`:
+Glue tables can be configured to contain user-defined properties, to describe the physical storage of table data, through the `storageParameters` property:
+
+```ts
+declare const myDatabase: glue.Database;
+new glue.Table(this, 'MyTable', {
+  storageParameters: [
+    glue.StorageParameter.skipHeaderLineCount(1), // note this is a number. The static method should convert it to a string.
```

Comment:
```suggestion
    glue.StorageParameter.skipHeaderLineCount(1),
```

Sorry the comment was just meant for you 😅

Reply:
Oh lol, I just changed it :)